csp_billing_adapter_google/plugin.py

```diff
@@ -75,6 +144,6 @@ def _fetch_metadata():
     try:
         value = urllib.request.urlopen(data_request).read()
     except urllib.error.URLError:
-        return None
+        return "{}"
```

Comment:
This will probably fix the exception that we were seeing being raised when GKE_METADATA was specified but the GKE & K8S service account plumbing wasn't being setup correctly...